libs/sdk-ui-tests-e2e/yarn.lock

```diff
@@ -0,0 +1,9460 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@aaronhayes/react-use-hubspot-form@^2.0.1":
```

Comment:
we do not use "yarn" in UI DK repo. Please double-check if it is necessary and move to npm